packages/server/graphql/mutations/helpers/addSeedTasks.ts

```diff
@@ -8,20 +11,35 @@ import generateUID from '../../../generateUID'
 const CONTENT_STRING = `
   This is a task card. They can be created here, in a meeting, or via an integration`
 
-const SEED_TASKS = [
-  {
-    status: 'active' as TaskStatusEnum,
-    sortOrder: 0,
-    content: convertToTaskContent(CONTENT_STRING),
-    plaintextContent: CONTENT_STRING
-  }
-]
+function getSeedTasks(teamId: string) {
+  const integrationURL = makeAppURL(appOrigin, `team/${teamId}/settings`)
+
+  const taskContent = `Parabol supports integrations for Jira, GitHub, GitLab, Slack and Mattermost. Connect your tools on Settings > Integrations.`
+  const taskContentHTML = `Parabol supports integrations for Jira, GitHub, GitLab, Slack and Mattermost. Connect your tools on <a href="${integrationURL}">Settings > Integrations</a>.`
+
+  const content = convertHtmlToTaskContent(taskContentHTML)
```

Comment:
-1 the names of the variables could be improved to distinguish them in which task they will be used. This also includes `CONTENT_STRING`.

Reply:
also updated, could you please help to review again, 😁